crates/types/src/config/admin.rs

```diff
@@ -157,6 +134,13 @@ impl AdminOptions {
             );
         }
     }
+    #[allow(deprecated)]
+    pub fn print_deprecation_warnings(&self) {
+        if self.query_engine.pgsql_bind_address.is_some() {
+            print_warning_deprecated_config_option("admin.query-enging.pgsql-bind-address", None);
+            eprintln!("Please make SQL queries using the CLI or with the :9070/query API.")
```

Comment:
Nit: it might not be 9070 depending on configuration 😃 Maybe we can just say admin API endpoint?

Reply:
Ah good catch.